debian/control

```diff
@@ -30,6 +30,7 @@ Depends:
  python3-sqlalchemy-utils,
  python3-stevedore,
  python3-yaml,
+ rabbitmq-server,
```

Comment:
I don't think we should add this depend, rabbitmq can be installed somewhere else.

Reply:
This will do thing, we have to undo in ansible for distributed system.

Reply:
Yeah I agree it is ugly, I don't like this fix neither (I maybe should remove the `--tags=mergeit` from my `git pr` alias :laughing: )
What will be the best solution?
And an acceptable solution?

Reply:
After discussion with @fredericlepied and @sduthil:
Best solution should be to manage it via ansible, but since we do not use ansible for upgrade procedure, it difficult to migrate it now.
Then the acceptable solution is just to remove the `debian-packaging` job from this repo :sweat_smile: